app/assets/javascripts/analytics/_events.js

```diff
@@ -89,7 +89,7 @@
 
   GOVUK.GDM.analytics.LinkClick = LinkClick;
 
-  GOVUK.GDM.analytics.events = {
+  GOVUK.GDM.analytics.buyerSpecificEvents = {
```

Comment:
I think the name here is supposed to correspond to the module/file name, to make understanding/debugging more sane?

Reply:
@galund - updated